ui-components/__tests__/actions/ExpandableContent.test.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react"
```

Comment:
🎊

Reply:
Could you explain more? :) React is necessary here.

Reply:
Yeah! Sorry it was a celebration emoji for the file :)